build.gradle

```diff
@@ -423,6 +423,10 @@ bootJar {
   }
 }
 
+jar {
+  enabled = false
```

Comment:
this was done as spring 2.5.x creates a plain-jar file was well as the actual jar file. we can do this or hard code the repo name in the docker file:
https://github.com/hmcts/hmc-hmi-inbound-adapter/blob/3ad93e1e9e9f63a3c44d17b8d3989925f7413d6f/Dockerfile#L4